decidim-core/app/services/decidim/action_authorizer.rb

```diff
@@ -29,37 +29,26 @@ def initialize(user, action, component, resource)
     def authorize
       raise AuthorizationError, "Missing data" unless component && action
 
-      status_code, data = if authorization_handler_name
-                            authorization_handler.authorize(authorization, permission_options, component, resource)
-                          else
-                            [:ok, {}]
-                          end
-
-      AuthorizationStatus.new(status_code, authorization_handler, data)
+      AuthorizationStatusCollection.new(authorization_handlers, user, component, resource)
     end
 
     private
 
     attr_reader :user, :component, :resource, :action
 
-    def authorization
-      return nil unless user && authorization_handler_name
-
-      @authorization ||= Verifications::Authorizations.new(organization: user.organization, user: user, name: authorization_handler_name).first
-    end
-
-    def authorization_handler
-      return unless authorization_handler_name
-
-      @authorization_handler ||= Verifications::Adapter.from_element(authorization_handler_name)
-    end
+    def get_authorization(handler_name)
```

Comment:
This has been renamed but I can't find where it's used

Reply:
You are right, after a redesign of `AuthorizationStatusCollection` this method was not longer required. Deleted.